GenderPayGap.WebUI/GenderPayGap.WebUI.csproj

```diff
@@ -96,7 +96,8 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
 
     <PackageReference Include="GovukNotify" Version="4.0.0" />
-
+    <PackageReference Include="JWT" Version="7.3.1" />

```

Comment:
the build was failing due to missing packages. I don't know how it has worked before because `GovukNotify 4.0.0 depends on JWT (>= 7.0.0 && < 8.0.0)` and JWT can't be found. 
(My assumption is the GovukNotify package included the dependency and it has been removed afterwards?)